libnavui-dropin/api/current.txt

```diff
@@ -0,0 +1,572 @@
+    method public void setInfoPanelTripProgressBinder(com.mapbox.navigation.dropin.binder.UIBinder? p);
+    method public void setManeuverBinder(com.mapbox.navigation.dropin.binder.UIBinder? p);
+    method public void setRoadNameBinder(com.mapbox.navigation.dropin.binder.UIBinder? p);
+    method public void setSpeedLimitBinder(com.mapbox.navigation.dropin.binder.UIBinder? p);
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? actionButtonsBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? infoPanelContentBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? infoPanelHeaderBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? infoPanelTripProgressBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? maneuverBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? roadNameBinder;
+    property public final com.mapbox.navigation.dropin.binder.UIBinder? speedLimitBinder;
+  }
+
+  public final class ViewOptionsCustomization {
+    ctor public ViewOptionsCustomization();
+    method public String? getMapStyleUriDay();
+    method public String? getMapStyleUriNight();
+    method public com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? getRouteArrowOptions();
+    method public com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? getRouteLineOptions();
+    method public void setMapStyleUriDay(String? p);
+    method public void setMapStyleUriNight(String? p);
+    method public void setRouteArrowOptions(com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? p);
+    method public void setRouteLineOptions(com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? p);
+    property public final String? mapStyleUriDay;
+    property public final String? mapStyleUriNight;
+    property public final com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions? routeArrowOptions;
+    property public final com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions? routeLineOptions;
+    field public static final com.mapbox.navigation.dropin.ViewOptionsCustomization.Companion Companion;
+  }
+
+  public static final class ViewOptionsCustomization.Companion {
+    method public com.mapbox.navigation.ui.maps.route.arrow.model.RouteArrowOptions defaultRouteArrowOptions(android.content.Context context);
+    method public com.mapbox.navigation.ui.maps.route.line.model.MapboxRouteLineOptions defaultRouteLineOptions(android.content.Context context);
+  }
+
+}
+
+package com.mapbox.navigation.dropin.binder {
+
+  public interface Binder<T> {
+    method public com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver bind(T? viewGroup);
+  }
+
+  public fun interface UIBinder extends com.mapbox.navigation.dropin.binder.Binder<android.view.ViewGroup> {
```

Comment:
@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)

This should not OptIn, remove the OptIn so it can be an experimental preview

Reply:
This is also a problem for other classes, like `NavigationView`. We'll need to refactor this where needed across the drop-in package.

Reply:
Yeah I have fixed it in all the classes. They are not `OptIn` anymore. Let me know if you think there are still some missing use cases. @LukasPaczos